packages/docs/.vitepress/components/VueMasteryLogoLink.vue

```diff
@@ -0,0 +1,84 @@
+<script setup lang="ts">
+const props = defineProps<{
+  for: string,
+}>()
+
+const links = {
+  'vue-router-cheat-sheet': 'https://www.vuemastery.com/vue-router/?coupon=ROUTER-DOCS&via=eduardo'
+}
+
+const link = links[props.for]
+
+const forVueRouterCheatSheet = props.for === 'vue-router-cheat-sheet'
+</script>
+
+<template>
+  <a :href="link" target="_blank">
+    <span class="logo-wrapper">
+      <img alt="Vue Mastery Logo" width="25px" src="https://firebasestorage.googleapis.com/v0/b/vue-mastery.appspot.com/o/flamelink%2Fmedia%2Fvue-mastery-logo-small.png?alt=media&token=941fcc3a-2b6f-40e9-b4c8-56b3890da108">
+    </span>
+    <span v-if="forVueRouterCheatSheet" class="description">
```

Comment:
that way, this becomes 
```suggestion
    <span v-if="description" class="description">
```